aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp

```diff
@@ -838,15 +832,11 @@ mha_bwd(const at::Tensor &dout,  // batch_size x seqlen_q x num_heads, x head_si
     #endif
     if (is_causal) { window_size_right = 0; }
     auto dprops = at::cuda::getCurrentDeviceProperties();
-    // bool is_sm75 = dprops->major == 7 && dprops->minor == 5;
-    bool is_sm8x = dprops->major == 8 && dprops->minor >= 0;
-    bool is_sm80 = dprops->major == 8 && dprops->minor == 0;
-    bool is_sm90 = dprops->major == 9 && dprops->minor == 0;
-    bool is_sm10x = dprops->major == 10 && dprops->minor >= 0;
-    bool is_sm120_or_sm121 = dprops->major == 12 && dprops->minor <= 1;
-    TORCH_CHECK(is_sm120_or_sm121 || is_sm10x || is_sm90 || is_sm8x, "FlashAttention only supports Ampere GPUs or newer.");
+    // bool is_sm75_or_newer = (dprops->major * 10 + dprops->minor) >= 80;
```

Comment:
can you check `test_transformers` to ensure that this is also the same there

Reply:
Yep, `test_transformers.py` checks `PLATFORM_SUPPORTS_FLASH_ATTENTION` variable, which is basically a `SM80OrLater` in CUDA.

https://github.com/pytorch/pytorch/blob/ead80f3202b23ad16daa3a250754ddb91d64e9f8/torch/testing/_internal/common_cuda.py#L58-L66